### ISSUE TITLE
Images and project titles only in home page gallery

### ISSUE THREAD

**Issue by kwritenow** [kwritenow/kristileach.com#18]
No text preview

**Comment by evz** [comment]
I'm noticing that you have thumbnails in the Behance gallery that aren't full sized images on the detail page for the project. So, for instance the Customer Portal Redesign has nice visualization thingy but it's not on the detail page. Any chance I can get larger versions of those thumbnails?

**Comment by kwritenow** [comment]
Yes. I have them but need to get my posts up for Allstate etc first
